landing_page/templates/landing_page/index.html

```diff
@@ -3,10 +3,9 @@
 <div id="message_div">
   <h1>Welcome to ReeldIn</h1>
 </div>
-<div id="content_div">
+<div class="content_div">
   <div id="title_div">
     <h2>Latest Recommendations...</h2>
-    <button id="get_recommendations_btm">Get Recommendations</button>
```

Comment:
why is this button getting axed? i'm using it in my branch.
if there was relocation i missed, let me know so i can place the functionality elsewhere!

Reply:
Yeah, the button is being moved to the header next to the login button/ profile pic when users are logged in. I thought that it looked better and can be seen on all pages now (without repeated code).